leftwm-core/src/handlers/window_handler.rs

```diff
@@ -439,10 +439,10 @@ pub fn scratchpad_xyhw(xyhw: &Xyhw, scratch_pad: &ScratchPad) -> Xyhw {
 fn sane_dimension(config_value: Option<Size>, default_percent: f32, max_pixel: i32) -> i32 {
     match config_value {
         Some(size) => match size {
-            Size::Percentage(percentage) if (0.0..0.9).contains(&percentage) => {
+            Size::Percentage(percentage) if (0.0..=1.0).contains(&percentage) => {
                 size.into_absolute(100.0) as i32 * max_pixel / 100
```

Comment:
Unless I'm misunderstanding the code, I think this can be changed as follows:
```suggestion
                size.into_absolute(max_pixel as f32) as i32
```

I know you didn't add this line, but I just saw that. :)

Reply:
Good catch